scripts/ttSF_purity.py

```diff
@@ -0,0 +1,398 @@
+import modules.ConfigReader as cfgr
```

Comment:
It's not entirely clear to me what this script is used for

Reply:
It is not clear because by mistake I added the wrong file (a quite complex git merge gone wrong...)! Thank you very much for noticing! The new file should be way more clear.